cypress/e2e/shared/buckets.test.ts

```diff
@@ -416,4 +420,73 @@ describe('Buckets', () => {
       })
     })
   })
+
+  describe('upload csv', function() {
+    it('can write a properly annotated csv', () => {
+      // Navigate to csv uploader
+      cy.get('@org').then(({id: orgID}: Organization) => {
+        cy.fixture('routes').then(({orgs, buckets}) => {
+          cy.get('@bucket').then(({id: bucketID}: Bucket) => {
+            cy.visit(`${orgs}/${orgID}${buckets}/${bucketID}/csv-uploader/new`)
+            cy.location('pathname').should(
+              'contain',
+              `${orgs}/${orgID}${buckets}/${bucketID}/csv-uploader/new`
+            )
+          })
+
+          // wait for animation to complete
+          cy.wait(2000)
```

Comment:
Is there anything visible we can interact with instead of a hard coded 2s wait?  It looks like if there's something we could have cypress `.get` that is animating the get function will wait for the animation to complete.

Reply:
I tried `.get`ing on a couple elements but couldn't get cypress to wait long enough for the component to fully render lol. It kept dragging and dropping the csv before the popup was even visible 😆  Oh you know I did have this idea yesterday that maybe it's because I'm navigating via the URL rather to the uploader rather than clicking the "add data" button. Let me try that approach and see if it helps at all

Reply:
woo got it to work!